samples/Ring/hello.ring

```diff
@@ -0,0 +1 @@
+See "Hello, World"
```

Comment:
The sample files are used for tests, but also to train the Bayesian classifier. We therefore prefer sample files from "real" applications (under permissive license).